BUILD_VERSIONS

```diff
@@ -2,4 +2,5 @@
 export SYNAPSE_VERSION=v1.5.1
 export POSTGRES_VERSION=10
 export NGINX_VERSION=1.17
-export RAIDEN_VERSION=v0.200.0-rc2
+export RAIDEN_VERSION=0.100.5a1.dev1577-ge9f080b7e.d20200305
```

Comment:
It is not a downgrade rather than the new version number corresponds to the output of `raiden version`. Since 0.100.5 was the last official release. It definitely looks a bit confusing, but it is the official version number of the most recent commit. Anyway it is just a tag for a specific raiden version which includes changes in the raiden code but is needed in the RSB master.